server/lib/transactions.js

```diff
@@ -242,3 +247,126 @@ export async function sum(where) {
   const result = await models.Transaction.findOne({ attributes, where });
   return result.dataValues.total;
 }
+
+const kindStrings = {
+  ADDED_FUNDS: `Added Funds`,
+  BALANCE_TRANSFER: `Balance Transfer`,
+  CONTRIBUTION: `Contribution`,
+  EXPENSE: `Expense`,
+  HOST_FEE: `Host Fee`,
+  HOST_FEE_SHARE: `Host Fee Share`,
+  HOST_FEE_SHARE_DEBT: `Host Fee Share Debt`,
+  PAYMENT_PROCESSOR_COVER: `Cover of Payment Processor Fee`,
+  PLATFORM_TIP: `Platform Tip`,
+  PLATFORM_TIP_DEBT: `Platform Tip Debt`,
+  PREPAID_PAYMENT_METHOD: `Prepaid Payment Method`,
+};
+
+export async function generateDescription(transaction, { req = null, full = false } = {}) {
+  let baseString = 'Transaction',
+    debtString = '',
+    tierString = '',
+    expenseString = '',
+    fromString = '',
+    toString = '';
+
+  if (transaction.isRefund && transaction.RefundTransactionId) {
+    const refundedTransaction = await (req
+      ? req.loaders.Transaction.byId.load(transaction.RefundTransactionId)
+      : models.Transactions.findByPk(order.RefundTransactionId));
+    if (refundedTransaction) {
+      const refundedTransactionDescription = await generateDescription(refundedTransaction, { req, full });
+      return `Refund of \`${refundedTransactionDescription}\``;
```

Comment:
Do we need to use backticks here? 
```suggestion
      return `Refund of "${refundedTransactionDescription}"`;
```

Reply:
Good question, my experience is that double quotes are sometime to be expected (in an Expense description for example, `SELECT * FROM "Transactions"  WHERE "description" ILIKE '%"%' AND "isRefund" IS FALSE`) and I was trying to find something that was not double quotes or single quotes.

Even if it's complicated, perhaps we could use backticks only if single quotes and double quotes are already used, JavaScript style. Or use a different syntax without quotes, "Refund: xxxx' or "Refund - xxxxx".